ur_documentation/docs/source/examples/my_robot_cell/start_ur_driver.rst

```diff
@@ -0,0 +1,115 @@
+    :language: py
+    :lineno-start: 57
+    :lines: 57 - 90
+    :linenos: 
+    :caption: my_robot_cell_control/launch/start_robot.launch.py
+
+In the following parts we need to create all **relevant Nodes** to get the driver started.
+
+We should begin by creating the **control node**, which requires our URDF, the update rate, and the ROS2 controllers file as parameters.
+
+.. literalinclude:: ../../../../../my_robot_cell/my_robot_cell_control/launch/start_robot.launch.py
+    :language: py
+    :lineno-start: 93
+    :lines: 93 - 102
+    :linenos: 
+    :caption: my_robot_cell_control/launch/start_robot.launch.py
+
+
+It's also a good idea to start the **controller stopper** node, which can stop and restart the ROS2 controllers as needed. 
+To function properly, we should specify the **consistent controllers** that must run continuously and should not be stopped by this node.
+
+
+
+.. literalinclude:: ../../../../../my_robot_cell/my_robot_cell_control/launch/start_robot.launch.py
+    :language: py
+    :lineno-start: 104
+    :lines: 104 - 121
+    :linenos: 
+    :caption: my_robot_cell_control/launch/start_robot.launch.py
+
+
+The **dashboard node** needs the robot ip to be started. 
+
+.. literalinclude:: ../../../../../my_robot_cell/my_robot_cell_control/launch/start_robot.launch.py
+    :language: py
+    :lineno-start: 123
+    :lines: 123 - 129
+    :linenos: 
+    :caption: my_robot_cell_control/launch/start_robot.launch.py
+
+The **robot state publisher** needs the urdf to be started.
+
+.. literalinclude:: ../../../../../my_robot_cell/my_robot_cell_control/launch/start_robot.launch.py
+    :language: py
+    :lineno-start: 132
+    :lines: 132 - 138
+    :linenos: 
+    :caption: my_robot_cell_control/launch/start_robot.launch.py
+
+The last nodes we need to start are the **controller spwaners**.
+
+.. literalinclude:: ../../../../../my_robot_cell/my_robot_cell_control/launch/start_robot.launch.py
+    :language: py
+    :lineno-start: 140
+    :lines: 140 - 149
+    :linenos: 
+    :caption: my_robot_cell_control/launch/start_robot.launch.py
+
+This is a small usefull function to **spawn a controller** and set an inactive flag if needed.
+
+Once we've determined which nodes we want to start, we can return the launch description with all arguments and nodes.
+
+.. literalinclude:: ../../../../../my_robot_cell/my_robot_cell_control/launch/start_robot.launch.py
+    :language: py
+    :lineno-start: 151
+    :lines: 151 - 166
+    :linenos: 
+    :caption: my_robot_cell_control/launch/start_robot.launch.py
+
+Remember that we haven't actually invoked the controller spawner node yet, so be sure to call the function for the ROS2 controllers you want to use.
+
+We can start the driver by running:
+
+.. code-block:: bash
+
+    #start the driver 
+    ros2 launch my_robot_cell_control start_robot.launch.py ur_type:=ur20 robot_ip:=yyy.yyy.yyy.yyy
```

Comment:
Let's add a section where we start ursim and specify the IP address of the default URSim interface.

```suggestion
    ros2 launch my_robot_cell_control start_robot.launch.py ur_type:=ur20 robot_ip:=192.168.56.101
```